app/src/main/java/com/cobresun/factfulnewsandroid/ui/activities/MainActivity.kt

```diff
@@ -31,10 +31,10 @@ class MainActivity : AppCompatActivity() {
         // Each page is represented by its own fragment.
         val viewPager = findViewById<ViewPager>(R.id.pager)
 
-        var adapter: TabsPagerAdapter
-        adapter = TabsPagerAdapter(supportFragmentManager, tabLayout.tabCount)
+        viewPager.adapter = TabsPagerAdapter(supportFragmentManager, tabLayout.tabCount)
+
+        viewPager.offscreenPageLimit = 0
```

Comment:
Whats the argument for changing this? Leaving it at default allows for smoother tab switching doesn't it?

Reply:
true i just left that in by accident when i was testing something. Fun fact tho, apparently setting it to 0 changes nothing.

Reply:
Why do we still have this then ?!